presto-main/src/test/java/com/facebook/presto/security/TestFileBasedSystemAccessControl.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.security;
+
+import com.facebook.presto.metadata.QualifiedObjectName;
+import com.facebook.presto.spi.CatalogSchemaName;
+import com.facebook.presto.spi.SchemaTableName;
+import com.facebook.presto.spi.security.AccessDeniedException;
+import com.facebook.presto.spi.security.Identity;
+import com.facebook.presto.transaction.TransactionManager;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.security.Principal;
+import java.util.Optional;
+import java.util.Set;
+
+import static com.facebook.presto.spi.security.Privilege.SELECT;
+import static com.facebook.presto.transaction.TransactionBuilder.transaction;
+import static com.facebook.presto.transaction.TransactionManager.createTestTransactionManager;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.fail;
+
+public class TestFileBasedSystemAccessControl
+{
+    private static final Principal PRINCIPAL = new TestingPrincipal("principal");
+    private static final Identity alice = new Identity("alice", Optional.of(PRINCIPAL));
+    private static final Identity bob = new Identity("bob", Optional.of(PRINCIPAL));
+    private static final Identity admin = new Identity("admin", Optional.of(PRINCIPAL));
+    private static final Identity nonAsciiUser = new Identity("ÎÎÎ", Optional.of(PRINCIPAL));
```

Comment:
don't write unicode characters directly, use the \u values (nice catch though, to be testing non-ascii)